admin/etc/empty.sql

```diff
@@ -0,0 +1,45 @@
+DELETE FROM `my_config_tree_group`;
```

Comment:
This config looks too opinionated to be named `empty.sql`. It has focus on *services* whereas README.md has guidelines (yes, opinionated too) to focus on a general logic instead. It is implied that `/onlineconf/chroot/*/*` is used for a services level.
I'd prefer this file to be explicitly marked as an example through naming.

Reply:
Your truth.
I renamed file.